fov_processing_pipeline/bin/process.py

```diff
@@ -60,10 +60,17 @@ def main():
         help="Save directory for results",
     )
     p.add_argument(
-        "--trim_data",
-        type=int,
-        default=10,
-        help="Trim cleaned data to this number (int) of fov's per cell line.",
+        "--dataset",
+        type=str,
+        default='quilt',
+        help="Which dataset to use, current can be \"quilt\", or \"labkey\"",
+    )
+
+    p.add_argument(
```

Comment:
Setting the default number of FoVs to 100. Sounds good!

Reply:
This seems to replace the trim data argument, where the previous default number of FoVs was 10

Reply:
I believe this was because "trim_data" sounds like it should be a boolean but was actually written to set the number of FOVs, so this is a little clearer. Is that what you were going for @gregjohnso ?

Reply:
yep